tests/nodeos_irreversible_mode_test.py

```diff
@@ -90,9 +90,12 @@ def getHeadLibAndForkDbHead(node: Node):
    forkDbHead =  int(info["fork_db_head_block_num"])
    return head, lib, forkDbHead
 
-# Around 30 seconds should be enough to advance lib for 4 producers
+# Wait for some time until LIB advance
 def waitForBlksProducedAndLibAdvanced():
-   time.sleep(30)
+   # Give 6 seconds buffer time
+   requiredConfirmation = int(2 / 3 * numOfProducers) + 1
+   timeToWait = ((12 * requiredConfirmation - 1) * 2) + 6
```

Comment:
This seems excessive. Remember that if you are calculated the amount of blocks to wait, you need to divide by 2 to get approximately the number of seconds to wait.

Also, I think you can even get away with the following amount of seconds: `12 * (requiredConfirmation - 1) + buffer_time_in_seconds`

Reply:
Right. I totally forgot dividing it by two here. Thanks for catching it